static/sass/styles.scss

```diff
@@ -1,7 +1,33 @@
 @import 'vanilla-framework/scss/build';
 
+$color-brand: #7C355D;
+
 @import 'pattern_icons';
+@import 'pattern_strip';
```

Comment:
I think the convention in other projects has been to alphabetise these `@include` and `@import` statements